applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_mohr_coulomb_with_tension_cutoff.cpp

```diff
@@ -188,30 +187,15 @@ KRATOS_TEST_CASE_IN_SUITE(MohrCoulombWithTensionCutOff_CalculateMaterialResponse
     // Arrange
     auto law = MohrCoulombWithTensionCutOff(std::make_unique<PlaneStrain>());
 
+    const auto                  properties = CreateProperties(35.0, 10.0, 0.0, 10.0);
     ConstitutiveLaw::Parameters parameters;
-    Properties                  properties;
-    properties.SetValue(GEO_FRICTION_ANGLE, 35.0);
-    properties.SetValue(GEO_COHESION, 10.0);
-    properties.SetValue(GEO_DILATANCY_ANGLE, 0.0);
-    properties.SetValue(GEO_TENSILE_STRENGTH, 10.0);
     parameters.SetMaterialProperties(properties);
-    ProcessInfo process;
-
-    Geometry<Node> dummyGeometry;
-    Vector         dummyVector;
-    law.InitializeMaterial(properties, dummyGeometry, dummyVector);
+    law.InitializeMaterial(properties, {}, {});
```

Comment:
Perhaps we can give the default-constructed objects a name, since it is unclear what they represent without looking at the function signature of `InitializeMaterial`, e.g.:
```suggestion
    const auto dummy_element_geometry = Geometry<Node>{};
    const auto dummy_shape_function_values = Vector{};
    law.InitializeMaterial(properties, dummy_element_geometry, dummy_shape_function_values);
```

Reply:
reverted